config/make-certs.py

```diff
@@ -128,6 +133,9 @@ def main():
         '--ca-key', cr.ca_key_file]
         + node_addresses)
 
+    subprocess.check_call(['rm', os.path.join(cr.node_certs_dir, 'ca.crt')])
```

Comment:
Maybe os.remove instead of shelling out?

Reply:
done